src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -93,45 +118,82 @@ const putS3ObjectBasic = async (
 
 const putS3Object = callWithRetryAndTimeout(putS3ObjectBasic);
 
-const deleteS3ObjectBasic = async (
-  object: S3Object
-): Promise<DeleteObjectCommandOutput> => {
+const deleteS3ObjectsBasic = async (
+  params: deleteS3ObjectByPrefixes
+): Promise<DeleteObjectsCommandOutput> => {
   if (runViaLambda())
     return (await sendLambdaCommand(
-      IntTestHelpers.deleteS3Object,
-      object
-    )) as unknown as DeleteObjectCommandOutput;
-
-  const bucketParams = {
-    Bucket: object.bucket,
-    Key: object.key,
+      IntTestHelpers.deleteS3Objects,
+      params
+    )) as unknown as DeleteObjectsCommandOutput;
+  const result: DeleteObjectsCommandOutput = {
```

Comment:
I doubt any function will actually use the return value and do something with the errors, but if you want to be thorough and return them, use your own datatype. We should stop bleeding the AWS-types into our own code. It's ports and adapters style to define your own internal interfaces and that's where we wanted to get to.